### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by OlivierBondu**
partly adresses #17

### CODE REVIEW

File: interface/HHAnalyzer.h [hunk 1]
```diff
@@ -131,6 +143,23 @@ class HHAnalyzer: public Framework::Analyzer {
         std::string m_electron_loose_wp_name;
         std::string m_electron_tight_wp_name;
 
+        // utilities
+        float getCosThetaStar_CS(LorentzVector h1, LorentzVector h2, float ebeam = 6500)
```
Comment (blinkseb) [inline interface/HHAnalyzer.h]: You should pass a `const LorentzVector&` here to avoid a copy at each function call.

File: plugins/HHAnalyzer.cc [hunk 2]
```diff
@@ -157,49 +164,58 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
...
+        for (unsigned int ibb = 0; ibb < bb_p4.size(); ibb++)
         {
             LorentzVector llbb = ll_p4[ill] + bb_p4[ibb];
             llbb_p4.push_back(llbb);
             llbb_idx.push_back(std::make_pair(ill, ibb));
             llbb_DR.push_back(ROOT::Math::VectorUtil::DeltaR(ll_p4[ill], jj_p4[ibb]));
             llbb_DPhi.push_back(ROOT::Math::VectorUtil::DeltaPhi(ll_p4[ill], jj_p4[ibb]));
-            float minDR = 14000;
+            float minDR = 14000.;
+            float maxDR = 0.;
```
Comment (blinkseb) [inline plugins/HHAnalyzer.cc]: These 2 initializations seems useless since you directly assign these below.

File: interface/HHAnalyzer.h (hunk 1 above)
Comment (OlivierBondu) [inline interface/HHAnalyzer.h]: addressed by 33bf236

File: plugins/HHAnalyzer.cc (hunk 2 above)
Comment (OlivierBondu) [inline plugins/HHAnalyzer.cc]: fixed in ae9cbf0

**Pull request merged**